classes/Earnings.php

```diff
@@ -164,7 +164,7 @@ public function prepare_earning_data( $total_price, $course_id, $order_id, $orde
 					$fees_amount = ( $total_price * $fees_amount ) / 100;
 				}
 
-				$course_price_grand_total = $total_price - $fees_amount;
+				$course_price_grand_total = ( $total_price > $fees_amount ) ? $total_price - $fees_amount : 0;
```

Comment:
We can use the follwoing way:
`$course_price_grand_total =  max( $total_price - $fees_amount , 0)`;